debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -683,6 +683,14 @@ public static SnapshotLockingMode parse(String value, String defaultValue) {
             .withImportance(Importance.LOW)
             .withDescription("The timeout in seconds for the creation of the replication slot.");
 
+    public static final Field AUTOMATIC_FLUSH = Field.create("automatic.flush")
```

Comment:
I don't think we want this option configurable as it IMHO always breaks Debezium's semnatics.
@Naros WDYT?

Reply:
Updated so that the `PGReplicationStream` will always have automatic flush disabled, and removed the option.